test/e2e/case19_rep_policy_placement_test.go

```diff
@@ -230,12 +230,15 @@ var _ = Describe("Test replicated_policy controller and propagation", Ordered, S
 			clientHubDynamic, gvrPolicy, testNamespace+"."+case19PolicyName, "managed2", true, defaultTimeoutSeconds,
 		)
 		Expect(plc2).ToNot(BeNil())
-		afterString = utils.GetMetrics("controller_runtime_reconcile_total",
-			`controller=\"replicated-policy\"`,
-			`,result=\"success\"`,
-		)[0]
-		afterTotal, err = strconv.Atoi(afterString)
-		Expect(err).ShouldNot(HaveOccurred())
-		Expect(afterTotal - beforeTotal).Should(Equal(2))
+
+		Eventually(func(g Gomega) {
```

Comment:
Flacky issue

Reply:
I don't think an `Eventually()` resolves this flaky test. Isn't the Flake where it returns `3` instead of `2`?

Reply:
yes yes but it resolved after this

Reply:
Never seen after this